Error when trying to generate a Windows WPK

|wazuh-packages version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.3 | any | any | Ubuntu 20 |

When trying to generate a Windows WPK by means of using `generate_wpk_package.sh` as follows:

```bash
./generate_wpk_package.sh -t windows -b wpk_fail_v4.3-rc1 -d /tmp/wpk -k /root/certs -pn /root/pckgs/wazuh-agent-4.3.0-rc3_wpk.msi -p /tmp/wpk -o WindowsAgentFail.wpk
```

The package generation process fails with the following error:

```bash
   copying cffi/_cffi_include.h -> build/lib.linux-x86_64-3.5/cffi
    copying cffi/parse_c_type.h -> build/lib.linux-x86_64-3.5/cffi
    copying cffi/_embedding.h -> build/lib.linux-x86_64-3.5/cffi
    copying cffi/_cffi_errors.h -> build/lib.linux-x86_64-3.5/cffi
    running build_ext
    building '_cffi_backend' extension
    creating build/temp.linux-x86_64-3.5
    creating build/temp.linux-x86_64-3.5/c
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/build/python3.5-5Cvh2K/python3.5-3.5.3=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DUSE__THREAD -DHAVE_SYNC_SYNCHRONIZE -I/usr/include/ffi -I/usr/include/libffi -I/usr/include/python3.5m -c c/_cffi_backend.c -o build/temp.linux-x86_64-3.5/c/_cffi_backend.o
    c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
     #include <ffi.h>
                     ^
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-j__5gb7m/cffi/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-_zfyyxtv-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-j__5gb7m/cffi/
The command '/bin/sh -c apt-get update &&     apt-get -y install python git curl jq python3 python3-pip &&     pip3 install --upgrade cryptography==2.9.2 awscli' returned a non-zero code: 1
Unable to find image 'common_wpk_builder:latest' locally
docker: Error response from daemon: pull access denied for common_wpk_builder, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

The previous message is only a slice of the whole execution's output. Here is the full output: [wpk_output_win.txt](https://github.com/wazuh/wazuh-packages/files/7652158/wpk_output_win.txt).

We've tried removing the related docker's images and instances with the following commands:

```
docker system prune
docker rmi $(docker images -a -q)
```

But the result remains the same.

Best Regards,

Koremblum Nicolás Mariano

--- a/wpk/common/Dockerfile
+++ b/wpk/common/Dockerfile
@@ -1,7 +1,7 @@
 FROM debian:9
 
 RUN apt-get update && \
-    apt-get -y install python git curl jq python3 python3-pip && \
+    apt-get -y install python git curl jq python3 python3-pip libffi-dev && \
     pip3 install --upgrade cryptography==2.9.2 awscli
 
 ADD wpkpack.py /usr/local/bin/wpkpack

